docs/indexing.md

```diff
@@ -57,3 +57,18 @@ as deploy user, in `/opt/marc_liberaton/current`
 $ RAILS_ENV=production UPDATE_LOCATIONS=false SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging-reindex bin/rake liberate:full --silent >> /tmp/full_reindex_[YYYY-MM-DD].log 2>&1
 
 This step takes about 21 hours
+
+### Index Partner SCSB records
+
+If needed, pull the most recent SCSB full dump records into a dump file:
+
+- as deploy user, in `/opt/marc_liberaton/current`
+- `$ RAILS_ENV=production bundle exec rake scsb:import:full
+- It kicks off an import job
```

Comment:
can you add a ` at the end of line $ RAILS_ENV=production bundle exec rake scsb:import:full

Reply:
yes thanks for catching this -- done.